src/actions/customToast.tsx

```diff
@@ -0,0 +1,38 @@
+import { toast } from 'react-toastify';
+
+type type = 'default' | 'info' | 'success' | 'error';
+
```

Comment:
you could add \

const defaultOptions = {
        hideProgressBar,
        autoClose: 3000
}

then use spread ...